tests/workers/test_base_worker.py

```diff
@@ -245,6 +245,53 @@ def create_run_with_deployment_2(state):
     assert {flow_run.id for flow_run in submitted_flow_runs} == set(flow_run_ids[1:3])
 
 
+async def test_worker_with_priority_and_limit(
```

Comment:
I think this name will help future developers quickly understand the purpose of this test.
```suggestion
async def test_priority_trumps_lateness(
```